### PR TITLE
Setup typescript updated

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/FRMW-743

### LGTMs
- [x] Dev LGTM
- [ ] Dev+Vidyo LGTM
- [ ] QE LGTM

### CODE REVIEW

File: utils/rx-page-objects/src/rxDatePicker.page.ts [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+    /**
+     * @private
+     * @type {String}
+     * @description (get/set) Month value of the picker _calendar_.
+     * **Format:** `MMM` (e.g. "Apr", "May", "Jun")
+     */
+    private get month(): AccessorPromiseString {
+        return rxSelect.initialize(this.element(by.model('currentMonth'))).selectedOption.getText();
+    }
+    private set month(value: AccessorPromiseString) {
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxDatePicker.page.ts]: These weren't private before. Why the change?

**Comment by CITguy** [comment]
LGTM

File: utils/rx-page-objects/src/rxDatePicker.page.ts (hunk 1 above)
Comment (Droogans) [inline utils/rx-page-objects/src/rxDatePicker.page.ts]: Ugh. Let me track it down. I think this was done from another computer when my main one was getting fixed last month 😮.

Comment (parlarjb) [inline utils/rx-page-objects/src/rxDatePicker.page.ts]: Did you look into this?

Comment (Droogans) [inline utils/rx-page-objects/src/rxDatePicker.page.ts]: Yes, it got addressed in the related PR linked at the bottom of this review.
